docs/userguide/datafiles.txt

```diff
@@ -8,12 +8,28 @@ for data files distributed with a package is for use *by* the package, usually
 by including the data files in the package directory.
 
 Setuptools offers three ways to specify data files to be included in your
-packages.  First, you can simply use the ``include_package_data`` keyword,
-e.g.::
+packages.  ``include_package_data`` accepts all data files and directories
```

Comment:
I liked the format from below with whitespace separating the different options and making it much easier to see that there are essentially three options here. How about bringing that format back?

Reply:
I'm not sure I understand the format your are referring to. Do you mind clarifying a little bit?

Reply:
In the old code, where it says "In summary, the three options allow you to", it breaks out three options into their tradeoffs, adding visual cues to the options.